rfc/rfc-51/rfc-51.md

```diff
@@ -64,69 +65,72 @@ We follow the debezium output format: four columns as shown below
 
 Note: the illustration here ignores all the Hudi metadata columns like `_hoodie_commit_time` in `before` and `after` columns.
 
-## Goals
+## Design Goals
 
 1. Support row-level CDC records generation and persistence;
 2. Support both MOR and COW tables;
 3. Support all the write operations;
 4. Support Spark DataFrame/SQL/Streaming Query;
 
-## Implementation
+## Configurations
 
-### CDC Architecture
+| key                                                 | default  | description                                                                                                                                      |
+|-----------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| hoodie.table.cdc.enabled                            | `false`  | The master switch of the CDC features. If `true`, writers and readers will respect CDC configurations and behave accordingly.                    |
+| hoodie.table.cdc.supplemental.logging               | `false`  | If `true`, persist the required information about the changed data, including `before`. If `false`, only `op` and record keys will be persisted. |
+| hoodie.table.cdc.supplemental.logging.include_after | `false`  | If `true`, persist `after` as well.                                                                                                              |
```

Comment:
i think one config `hoodie.table.cdc.supplemental.logging` maybe is enough. Less configs will improve the ease of use. If not, `hoodie.table.cdc.supplemental.logging.include_before` will be needed to provide.

Reply:
The intention is, when users don't set any of these (everything default), cdc data will be minimized. And the more config turn on, more data will be logged. An enum config "logging.mode" as mentioned above does help simplify it.

Reply:
+1 for `hoodie.table.cdc.supplemental.logging.mode` that have these three values for now.

Reply:
Should we expose such parameters ? They looks confusing at first glance, if we expose them, a detailed instruction on they work for different use cases should be explained.

Reply:
> a detailed instruction on they work for different use cases should be explained.

Yes. as shown above, just 1 config `hoodie.table.cdc.supplemental.logging.mode` is to be exposed and the values indicating different level of logging verbosity should be explained in config docs. By default only op and key are to be logged, as the minimized logging mode. I've updated the PR for this accordingly.